src/ad-sizes.ts

```diff
@@ -38,6 +39,13 @@ class AdSize extends Array<number> {
 		return [this[0], this[1]];
 	}
 
+	// The inserted advert is a fixed size within its slot.
+	// Examples that are not a fixed size are: fluid ads, empty or
+	// out of page ads, and merch ads in a [88, ??] size format.
+	public isFixedSize(): boolean {
+		return this.width > 2 && this.width !== 88;
```

Comment:
While this is very smart I worry about the readability a little.

Could we do something like:

```suggestion
    const isOutOfPage = this.width === 1 && this.height === 1;
    const isFluid = this.toString() === 'fluid';
    const isProxy = this.width === 88;
		return !(isOutOfPage || isFluid || isProxy);
```

Reply:
Yeah this is much better 👍